ComfyUI/comfy_api_nodes/apis/client.py

```
@@ -248,17 +250,24 @@ def get_headers(self) -> Dict[str, str]:
+                headers["x-key"] = self.comfy_api_key
+        # Special case for Ideogram API
+        elif "api.ideogram.ai" in self.base_url and self.auth_token:
+            headers["Api-Key"] = self.auth_token
+        # Default behavior for other APIs
+        else:
+            if self.auth_token:
+                headers["Authorization"] = f"Bearer {self.auth_token}"
+            if self.comfy_api_key:
+                headers["x-key"] = self.comfy_api_key
+
+        print(f"[DEBUG] Final headers for {self.base_url}: {headers}")
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)